frontend/packages/process-editor/src/utils/hookUtils/hookUtils.ts

```diff
@@ -0,0 +1,55 @@
+import type { BpmnDetails } from '../../types/BpmnDetails';
+import type { BpmnBusinessObjectViewer } from '../../types/BpmnBusinessObjectViewer';
+import type { BpmnBusinessObjectEditor } from '../../types/BpmnBusinessObjectEditor';
+
+/**
+ * Gets the bpmn details from the business object in viewer mode
+ * @param businessObject the business object in viewer mode
+ * @returns the bpmn details
+ */
+export const getBpmnViewerDetailsFromBusinessObject = (
+  businessObject: BpmnBusinessObjectViewer,
+): BpmnDetails => {
+  console.log('businessobject', businessObject);
+  const bpmnType = businessObject?.$type;
+  const bpmnId = businessObject?.id;
+  const bpmnName = businessObject?.name;
+  const bpmnAttrs = businessObject?.$attrs;
+  const bpmnTasktype = bpmnAttrs ? bpmnAttrs['altinn:tasktype'] : null;
```

Comment:
Just a small comment on these variables - since they are not used anywhere else in this method, we could just as well skip the variables and set the values directly in the `bpmnDetails` object.

Reply:
Fixed 😄